src/chapters/databases/more-db-practice.rst

```diff
@@ -0,0 +1,150 @@
+More Database Practice
+======================
+
+In the previous pages, we created a database, added a table to it, and saved
+several entries in that table. Now we will learn how to modify the stored data.
+
+Update Table Data
+-----------------
+
+By using an :ref:`UPDATE <sql-update-syntax>` query, we can change one or more
+entries in a table. Code along with the following video to practice this
+technique.
+
+.. raw:: html
+
+   <section class="vid_box">
+      <iframe class="vid" src="https://www.youtube.com/embed/piXiY9j64I4" frameborder="1" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+   </section>
+
```

Comment:
There is an extra column and value in this video, (recent grad  yes) but the students haven't added that and might get stuck.  Maybe a short video about adding columns?

Reply:
Good catch!  I totally missed that.

I added an admonition box above the video that gives the students the code they need to run.  Let me know what you think.

Reply:
Looks Good!